### PR TITLE
Package updates

### PR DESCRIPTION
**Pull request by lrusak**
Just a bunch of package updates.

Only build tested not runtime tested.

### CODE REVIEW

**Comment by lrusak** [comment]
runtime test seems ok. @MilhouseVH care to include in the next build?

File: packages/sysutils/systemd/package.mk [hunk 1]
```diff
@@ -103,7 +103,8 @@ PKG_CONFIGURE_OPTS_TARGET="ac_cv_func_malloc_0_nonnull=yes \
                            --with-dbussessionservicedir=/usr/share/dbus-1/services \
                            --with-dbussystemservicedir=/usr/share/dbus-1/system-services \
                            --with-rootprefix=/usr \
-                           --with-rootlibdir=/usr/lib"
+                           --with-rootlibdir=/usr/lib \
+                           --with-default-hierarchy=hybrid"
```
Comment (escalade) [inline packages/sysutils/systemd/package.mk]: Docker needs legacy hierarchy, hybrid breaks it.

Comment (HiassofT) [inline packages/sysutils/systemd/package.mk]: it looks like the docker issues are already resolved - see comment here https://bugs.archlinux.org/task/53271
haven't tested but if this is the case better patch/update docker than staying on legacy cgroups v1

**Comment by MilhouseVH** [comment]
We're no longer building `hciattach` with this version of `bluez`

**Comment by lrusak** [comment]
@MilhouseVH updated with bluez commit

File: packages/sysutils/systemd/package.mk (hunk 1 above)
Comment (lrusak) [inline packages/sysutils/systemd/package.mk]: I'll just update docker

**Comment by MilhouseVH** [comment]
Ready to merge? No issues reported...

**Comment by lrusak** [comment]
Yep, updated with a commit to mesa 17.0.2

**Pull request merged**